business_requirement_deliverable/views/business_view.xml

```diff
@@ -321,34 +201,44 @@
                         </group>
                         <group>
-                            <label for="qty"/>
+                            <label for="qty" attrs="{'readonly':[('state','not in',['draft', 'confirmed'])]}"/>
                             <div>
-                                <field name="qty"/>
-                                <field name="uom_id" class="oe_inline"/>
+                                <field name="qty" attrs="{'readonly':[('state','not in',['draft', 'confirmed'])]}"/>
+                                <field name="uom_id" class="oe_inline" attrs="{'readonly':[('state','not in',['draft', 'confirmed'])]}"/>
                             </div>
-                            <field name="sale_price_unit" groups="business_requirement_deliverable.group_business_requirement_estimation"/>
-                            <field name="currency_id" groups="business_requirement_deliverable.group_business_requirement_estimation"/>
-                            <field name="price_total" widget="monetary" options="{'currency_field': 'currency_id'}" groups="business_requirement_deliverable.group_business_requirement_estimation"/>
+                            <field name="sale_price_unit" attrs="{'readonly':[('state','not in',['draft', 'confirmed'])]}"
+                                   groups="business_requirement_deliverable.group_business_requirement_estimation"  />
+                            <field name="currency_id" attrs="{'readonly':[('state','not in',['draft', 'confirmed'])]}"
+                                   groups="business_requirement_deliverable.group_business_requirement_estimation"/>
+                            <field name="price_total" attrs="{'readonly':[('state','not in',['draft', 'confirmed'])]}" widget="monetary"
+                                   options="{'currency_field': 'currency_id'}" groups="business_requirement_deliverable.group_business_requirement_estimation"/>
                         </group>
                     </group>
                     <group string="Resource Lines">
                         <field name="resource_ids"
                         context="{'tree_view_ref':'business_requirement_deliverable.view_business_requirement_resource_tree',
                         'default_business_requirement_id': business_requirement_id}"
                         nolabel="1">
-                        <tree editable="1">
-                            <field name="resource_type"/>
-                            <field name="product_id"/>
-                            <field name="name"/>
-                            <field name="user_id" attrs="{'readonly':[('resource_type','=','procurement')]}"/>
-                            <field name="qty"/>
-                            <field name="uom_id"/>
-                            <field name="business_requirement_id" invisible="1"/>
+                        <tree editable="1" >
```

Comment:
It's best to leave the Tree view defined it it's own record, rather than defining it inside a field - it makes it easier to extend.